### PR TITLE
Enabled rs guest and host to communicate via stdout

### PR DESCRIPTION
**Pull request by ritazh**
Signed-off-by: Rita Zhang <rita.z.zhang@gmail.com>

### CODE REVIEW

File: privileged/Makefile [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 	# cargo build --target=wasm32-wasi --release
 	# cp target/wasm32-wasi/release/*.wasm policy.wasm
-	cargo build --release --target wasm32-unknown-unknown
-	cp target/wasm32-unknown-unknown/release/*.wasm policy.wasm
+	cargo build --release --target wasm32-wasi
+	cp target/wasm32-wasi/release/*.wasm policy.wasm
+
+.PHONY: run
+run:
+	go get github.com/tetratelabs/wazer
```
Comment (sozercan) [inline privileged/Makefile]: do we need this?

File: privileged/src/lib.rs [hunk 2]
```diff
@@ -89,21 +89,21 @@ unsafe fn deallocate(ptr: *mut u8, size: usize) {
...
-    let object_to_test = r#"{"apiVersion":"v1","kind":"Pod","metadata":{"name":"nginx","labels":{"app":"nginx"}},"spec":{"containers":[{"name":"nginx","image":"nginx","securityContext":{"privileged":true}}]}}"#;
+    let object_to_test = r#"{"apiVersion":"v1","kind":"Pod","metadata":{"name":"nginx","labels":{"app":"nginx"}},"spec":{"containers":[{"name":"nginx","image":"nginx","securityContext":{"privileged":false}}]}}"#;
 
     match serde_json::from_str::<apicore::Pod>(object_to_test) {
         Ok(pod) => {
             if let Some(pod_spec) = &pod.spec {
                 return match validate_pod(pod_spec) {
-                    Ok(_) => log("true".to_string()),
-                    Err(_) => log("false".to_string()),
+                    Ok(_) => println!("true"),
```
Comment (sozercan) [inline privileged/src/lib.rs:98]: I was getting `strconv.ParseBool: parsing "false\n": invalid syntax` on wasm driver side, using `print` here didn't work

but using `strings.TrimSpace(string(decision))` in wasm driver works

Comment (ritazh) [reply privileged/src/lib.rs:98]: oh yea cuz println includes the `\n'

File: privileged/Makefile (hunk 1 above)
Comment (ritazh) [reply privileged/Makefile]: that should be wazero. we need it for host.go. repushed

**Pull request merged**